manim/default.cfg

```diff
@@ -127,6 +127,13 @@ save_last_frame = False
 save_pngs = False
 save_as_gif = False
 
+[custom_folders]
+media_dir = /home/jh/Downloads/temp_movie_files
+video_dir = /home/jh/Downloads/temp_movie_files
+tex_dir   = /home/jh/Downloads/temp_movie_files
+text_dir  = /home/jh/Downloads/temp_movie_files
+output_file = /home/jh/Downloads/XXX
```

Comment:
You don't need to specify the name of the output file, as manim will use the name of the Scene by default. So the XXX doesn't need to be specified here.

Reply:
Also not resolved yet, it gives me an error when I only  have output_file = %(custom_root)s/  :
`IsADirectoryError: [Errno 21] Is a directory: '/home/jan-hendrik/Videos/aux'`